virtcontainers/qemu.go

```diff
@@ -1553,6 +1567,23 @@ func (q *qemu) resizeVCPUs(reqVCPUs uint32) (currentVCPUs uint32, newVCPUs uint3
 	return currentVCPUs, newVCPUs, nil
 }
 
+func (q *qemu) updateMemoryBalloon(sizeMB uint32) error {
+
+	if q.state.hasVFIODevice {
+		//Reference:
+		//https: //www.ibm.com/support/knowledgecenter/en/SSZJY4_3.1.0/liabp/liabpmemoryballooning.htm
+		hostMB, err := q.hostMemMB()
+		if err != nil {
+			return err
+		}
+		sizeMB = uint32(hostMB)
+		q.Logger().Warnf("VM uses VFIO, balloon size will be total host memory (%d MB)", sizeMB)
```

Comment:
q.Logger().WithField("mem", sizeMB).Warn("VM uses VFIO, balloon size will be total host memory")